### PR TITLE
Implements fixed gradient stops to prevent the background color jumping around

### LINKED ISSUE #2298: Background brightness changes when changing tabs
There is this weird bug where switching between the Benchmarks and Overview tab very slightly changes the background brightness.

https://www.opendota.com/matches/5118002212

EDIT: it's on the right side

**Environment**
- Browser Version: chromium
- OS: win 10

### PR DESCRIPTION
**Pull request by bdbch**
closes #2298

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (howardchung) [inline .prettierrc]: we don't need this if we aren't using prettier?

File: src/components/App/GlobalStyle.jsx [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import { createGlobalStyle } from 'styled-components';
+
```
Comment (howardchung) [inline src/components/App/GlobalStyle.jsx]: don't think we need this line

**Comment by howardchung** [comment]
@bdbch were you able to resolve the comments?

**Comment by bdbch** [comment]
I'll close this one since @Argn0 already fixed this in the new color scheme PR (seems so).

I finally have more time for OS projects over the holidays so expect some more work from my side. :)

**Pull request closed without merging**